floors/floors.go

```diff
@@ -0,0 +1,89 @@
+package floors
+
+import (
+	"fmt"
+	"math"
+	"math/rand"
+
+	"github.com/mxmCherry/openrtb/v16/openrtb2"
+	"github.com/prebid/prebid-server/currency"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+const (
+	DEFAULT_DELIMITER      string = "|"
+	CATCH_ALL              string = "*"
+	SKIP_RATE_MIN          int    = 0
+	SKIP_RATE_MAX          int    = 100
+	MODEL_WEIGHT_MAX_VALUE int    = 100
+	MODEL_WEIGHT_MIN_VALUE int    = 0
+	ENFORCE_RATE_MIN       int    = 0
+	ENFORCE_RATE_MAX       int    = 100
```

Comment:
Nitpick: In Go, it's common to use CamelCase for constants instead of CAPS. ie DEFAULT_DELIMITER -> DefaultDelimiter. This is largely due to the capital letter at the start of the variable name indicating if the type is package private or exported. If no other package needs these constants, please start with a lower case letter.

Reply:
updated to use CamelCase